README.md

```diff
@@ -92,16 +77,21 @@ Both the indexer and the web service allow for project-specific customizations
 via a plug-in mechanism, allowing the Boardwalk UI codebase to be functionally
 generic with minimal need for project-specific behavior.
 
-# 1. Getting Started
+## 1.2 Architecture Diagram
 
-## 1.1 Development Prerequisites
+![Azul architecture diagram](docs/azul-arch.png)
 
-- Python 3.6 (3.7 does not work) with `pip`
+# 2. Getting Started
+
+## 2.1 Development Prerequisites
+
+- Python 3.6 (3.7 does not work) with `pip`. Micro versions 3.6.5 and 3.6.8 are 
+  known to work, but 3.6.9 does not.
```

Comment:
I'm using 3.6.9. What exactly does not work?

Reply:
Failure to install dependencies via pip due to SSL errors.

Reply:
The issue may be specific to Mac/OSX.

Reply:
Who has observed the problem under 3.6.9 on OS X? If it is specific to OS X, this fact should be stated here.  I would like this issue with 3.6.9 to be corroborated/demonstrated. If that's not possible, I would like to remove the clause from this sentence. If it is possible, I'd like a ticket to be created and a reference to the ticket put into the README right here. 3.6 is our target platform so it's a big deal if Azul does not work on the most recent 3.6 release.

Reply:
I encountered it on OSX (on the macbook I was loaned when I first started here). Downgrading to 3.6.5 fixed the issue on that computer. I haven't tried using 3.6.9 on any other platform, so I can't verify that it's specific.

Reply:
Did you install 3.6.9 on that loaner or was it already on there?

Reply:
I'm fairly certain I installed it. Only 2.7 was installed by default.